distribution/experimental/profiles/profile-resources/my-username/resources.yaml

```diff
@@ -0,0 +1,265 @@
+---
+apiVersion: kubeflow.org/v1beta1
+kind: Profile
+metadata:
+  name: my-username
+spec:
+  owner:
+    kind: User
+    name: my.username@mydomain.com
+  plugins:
+  - kind: AwsIamForServiceAccount
+    spec:
+      awsIamRole: arn:aws:iam::123456789012:role/my-cluster-merge_profile_my-username
+---
+apiVersion: kubeflow.org/v1alpha1
+kind: PodDefault
+metadata:
+  name: access-ml-pipeline
+  namespace: my-username
+spec:
+  desc: Allow access to Kubeflow Pipelines
+  selector:
+    matchLabels:
+      access-ml-pipeline: 'true'
+  volumeMounts:
+  - mountPath: /var/run/secrets/ml-pipeline
+    name: volume-ml-pipeline-token
+    readOnly: true
+  volumes:
+  - name: volume-ml-pipeline-token
+    projected:
+      sources:
+      - serviceAccountToken:
+          path: token
+          expirationSeconds: 7200
+          audience: pipelines.kubeflow.org
+  env:
+  - name: ML_PIPELINE_SA_TOKEN_PATH
+    value: /var/run/secrets/ml-pipeline/token
+---
+apiVersion: security.istio.io/v1beta1
+kind: PeerAuthentication
+metadata:
+  name: force-strict
+  namespace: my-username
+spec:
+  mtls:
+    mode: STRICT
+---
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: allow-knative-traffic
+  namespace: my-username
+spec:
+  action: ALLOW
+  rules:
+  - from:
+    - source:
+        namespaces: [knative-serving]
```

Comment:
This shouldn't be necessary since it is handled by the profile controller already. Or were more permissions necessary than the paths specified by the profile controller?

Reply:
Ah, I think this can be removed. Nice catch, this was something I added while debugging. Will remove and then confirm if it still works.